docs/MinimallyImpactfulServicing.md

```diff
@@ -0,0 +1,99 @@
+# Minimally Impactful Servicing of eBPF For Windows
+
+## Overview
+As newer versions of eBPF for Windows are released, it is necessary to replace
+the code currently executing on systems. Simply stopping and restarting eBPF is
+not a viable solution as applications are dependent on state stored within eBPF,
+in the form of map, program, and link objects. This document outlines a strategy
+for replacing the currently executing version of eBPF for Windows with a newer
+version.
+
+## Architectural changes
```

Comment:
Missing discussion of what happens to data (i.e., things that would result in ebpf programs being invoked) during an upgrade.  Is the data path halted?  Do things "leak" in terms of going through without any ebpf program invocation?

Reply:
I will make it explicit, and we can then revise based on feedback.